documentation/how-to/pg-service.en.md

```diff
@@ -23,3 +23,76 @@ Usually the path on Android devices looks something like this: `/Android/data/ch
 QFieldCloud support `pg_service.conf` configurations too. You need to configure your PostgreSQL layers with "Offline editing" cloud action and store your service settings on QFieldCloud Project's Secrets page.
 
 Read more [how to configure PostgreSQL service](../reference/qfieldcloud/secrets.md) in the QFieldCloud documentation.
+
+## Creating a `pg_service.conf` File for PostgreSQL Connection in QGIS and Secrets
+
+Before beginning, ensure that your PostgreSQL database allows connections from QFieldCloud. Refer to [Technical specs](../reference/qfieldcloud/specs.md) for instructions.
+
+### Setup `pg_service.conf` File
+
+We first need to setup a configuration file. There are a lot of option to organize this, [read more in the PostgreSQL documentation](https://www.postgresql.org/docs/current/libpq-pgservice.html) or follow the description below.
+
+1. **Create a Configuration File**:
+
+   - On Windows:
+     Create a file named `pg_service.conf` and store it in a convenient location.
+
+   - On Linux/MacOS/Unix:
+     Create a file named `.pg_service.conf` in your home folder (`~`).
+
+2. **Define Connection Parameters**:
+
+   Within the file, specify connection parameters for your PostgreSQL database using the following format:
+
+   ```
+   [SERVICE_NAME]
+   host=your_host_or_ip
+   port=your_port
+   dbname=your_database_name
+   user=your_username
+   password=your_password
+   ```
+
+   Replace placeholders (`your_host_or_ip`, `your_port`, `your_database_name`, `your_username`, `your_password`) with actual connection details.
+
+3. **Save the File**:
+   Save the file after adding the connection parameters.
+
+   ```
+   [NINJA_DB]
+   host=my_db_for_ninja.ch
+   port=5432
+   dbname=my_database
+   user=my_username
+   password=my_password
+   ```
+
+!![Parameters](../assets/images/service_config_file_001.png)
```

Comment:
I don't think this is really needed to be repeated twice, as well the blurred screenshot. Isn't it better if we remove 3.?

Reply:
I completely agree!

Reply:
I take new screeshots that looks a little bit better